boardcfg/ethernet_sk2/app_settings.h

```diff
@@ -141,6 +141,13 @@ extern "C" {
 #define RDM_RESPONDER_MUTE_PORT_BIT PORTS_BIT_POS_12
 
 /**
+ * @brief The pin to use for the RDM temperature sensor.
+ *
+ * If undefined, we use a 'dummy' sensor that generates random values.
```

Comment:
Aren't we better off not doing an alternative dummy sensor and having less sensors, so we can differentiate if it's working or not?